src/EventStore.Core/TransactionLog/Chunks/TFChunk/TFChunk.cs

```diff
@@ -1194,14 +1136,14 @@ private enum CacheStatus {
 
 			// if we get here, then we reserved TFChunk for sure so no one should dispose of chunk file
 			// until client returns dedicated reader
-			return new TFChunkBulkReader(this, GetSequentialReaderFileStream(), isMemory: false);
+			return new TFChunkBulkReader(this, CreateFileStreamForBulkReader(), isMemory: false);
 		}
 
-		private Stream GetSequentialReaderFileStream() {
-			return _inMem
-				? (Stream)new UnmanagedMemoryStream((byte*)_cachedData, _fileSize)
-				: new FileStream(_filename, FileMode.Open, FileAccess.Read, FileShare.ReadWrite, 65536,
-					FileOptions.SequentialScan);
+		private Stream CreateFileStreamForBulkReader() {
```

Comment:
the old code here was checking _inMem (but not _unbuffered), was this an intentional change?